dashboard/config/crd/kustomization.yaml

```
@@ -0,0 +1,21 @@
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
+# It should be run by config/default
+resources:
+- bases/addons.k8s.io_dashboards.yaml
```

Add setting for building manifests of Application CR because dashboard operator use it.

```suggestion
- bases/addons.k8s.io_dashboards.yaml
- bases/app.k8s.io_application.yaml
```